python/pyspark/rdd.py

```diff
@@ -465,6 +465,21 @@ def getCheckpointFile(self) -> Optional[str]:
 
         return checkpointFile.get() if checkpointFile.isDefined() else None
 
+    def cleanShuffleDependencies(self, blocking: bool = False) -> None:
+        """
+        Removes an RDD's shuffles and it's non-persisted ancestors.
+
+        When running without a shuffle service, cleaning up shuffle files enables downscaling.
+        If you use the RDD after this call, you should checkpoint and materialize it first.
+
+        .. versionadded:: 3.3.0
+
+        Notes
```

Comment:
Maybe we should also better add Parameters section (I know many of them don't have it here but should be best to have it)

Reply:
Oh also we should add it in the list at https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.rst

Reply:
Oh, thank you. I'll take a look at them.